### DISCUSSION TITLE
Any good solutions for polls?

### DISCUSSION THREAD

**Discussion by YetAnotherGerrit** [marp-team/marp#552] [Q&A]
Has someone of you found a good way to integrate polls into your marp presentations? Like with Slido, Strawpoll, Menimeter or something like that?

I guess, something like an iframe is needed, correct?

**Comment by yhatt** [comment]
The choice of a service for poll is left to the decision of the slide author. Marp team would not provide/promote any specific poll service. Please feel free to use a service that suits your needs.

For using `<iframe>` HTML elements within Markdown, you have to allow the use of HTML in Markdown in each Marp tool explicitly.

**Reply by YetAnotherGerrit** [reply]
Thank you! That was, what I am aiming for. And I can understand that you don't want to promote any specific server officially, maybe some marp users/community members have some recommendations?
